query-engine/connector-test-kit-rs/query-engine-tests/tests/new/ref_actions/on_update/cascade.rs

```diff
@@ -386,3 +386,125 @@ mod one2many_opt {
         Ok(())
     }
 }
+
+#[test_suite(schema(schema))]
+mod multiple_cascading_paths {
+    use indoc::indoc;
+    use query_engine_tests::run_query;
+
+    fn schema() -> String {
+        let schema = indoc! {
+            r#"model User {
+                #id(id, Int, @id)
+                idd Int @unique
```

Comment:
not a fan of the naming, something like `uniq` makes it clearer what is happening.